modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/dto/v2_0/util/DataDefinitionUtil.java

```diff
@@ -267,17 +266,12 @@ private static DataDefinitionField[] _toDataDefinitionFields(
 			return new DataDefinitionField[0];
 		}
 
-		Stream<DDMFormField> stream = ddmFormFields.stream();
-
-		return stream.map(
+		return TransformUtil.transformToArray(
```

Comment:
@JanisZhang Seems like it's the same case with https://github.com/brianchandotcom/liferay-portal/pull/130831